apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -1,277 +1,322 @@
+                                    Copy Schedule
+                                </Button>
+                                <Menu {...bindMenu(popupState)}>
+                                    {scheduleNames.map((name, index) => (
                                         <MenuItem
-                                            onClick={() => {
-                                                copySchedule(this.state.scheduleNames.length);
-                                                popupState.close();
-                                            }}
+                                            key={index}
+                                            disabled={AppStore.getCurrentScheduleIndex() === index}
+                                            onClick={createCopyHandler(index)}
                                         >
-                                            Copy to All Schedules
+                                            Copy to {name}
                                         </MenuItem>
-                                    </Menu>
-                                </>
-                            )}
-                        </PopupState>
-                        <Button
-                            className={this.props.classes.clearSchedule}
-                            variant="outlined"
-                            color="inherit"
-                            onClick={() => {
-                                if (
-                                    window.confirm(
-                                        'Are you sure you want to clear this schedule? You cannot undo this action, but you can load your schedule again.'
-                                    )
-                                ) {
-                                    clearSchedules();
-                                    logAnalytics({
-                                        category: analyticsEnum.addedClasses.title,
-                                        action: analyticsEnum.addedClasses.actions.CLEAR_SCHEDULE,
-                                    });
-                                }
-                            }}
-                        >
-                            Clear Schedule
-                        </Button>
-                    </div>
-                </div>
-                {this.state.courses.map((course) => {
+                                    ))}
+                                    <MenuItem onClick={createCopyHandler(scheduleNames.length)}>
+                                        Copy to All Schedules
+                                    </MenuItem>
+                                </Menu>
+                            </>
+                        )}
+                    </PopupState>
+
+                    <Button
+                        sx={{ marginLeft: '4px', marginRight: '4px' }}
+                        variant="outlined"
+                        color="secondary"
```

Comment:
```suggestion
                        color="inherit"
```
Currently invisible on light mode